www/scripts/codecheckerviewer/codecheckerviewer.js

```diff
@@ -29,6 +29,9 @@ function (declare, topic, domConstruct, Dialog, DropDownMenu, MenuItem,
 
     CC_SERVICE = new codeCheckerDBAccess.codeCheckerDBAccessClient(
       new Thrift.Protocol(new Thrift.Transport("CodeCheckerService")));
+    var transport = new Thrift.TXHRTransport("/Authentication");
+    var protocol  = new Thrift.TJSONProtocol(transport);
+    var CC_AUTH_SERVICE    = new codeCheckerAuthentication.codeCheckerAuthenticationClient(protocol);
```

Comment:
Does `TXHRTransport` differ from `Thrift.Transport`? What's `Thrift.Protocol` versus `Thrift.TJSONProtocol`? (I think stating simply protocol means default, which is either hard-coded to be JSON in terms of the browser (as CodeChecker works :grinning: ), or Thrift actually negotiates with the server on which protocol to use.

Nonetheless, if these variables are not used later on, I think we should simplify this initialisation into one line, as is the case with `CC_SERVICE`.